src/components/projectSelect/index.tsx

```diff
@@ -62,18 +90,27 @@ export const ProjectSelect: React.FC<ProjectSelectProps> = ({
   return (
     <SearchableDropdown
       label="Project"
-      value={selectedProject?.displayName || selectedProject?.identifier}
+      value={
+        selectedProject?.displayName ||
+        selectedProject?.identifier ||
+        selectedProjectIdentifier
+      }
       options={allProjects}
       onChange={(projectIdentifier: any) => {
```

Comment:
I know this was already here but can we type `projectIdentifier` as a string instead of any

Reply:
<img width="672" alt="image" src="https://user-images.githubusercontent.com/13104717/157731964-fbc2ddc4-c43a-412b-9796-c5807323a286.png">
string doesn't work as a type here

Reply:
Looks like we need some better typescript generic handling :) 
https://jira.mongodb.org/browse/EVG-16516

Reply:
:)